Misc/NEWS.d/next/Core and Builtins/2018-04-25-20-44-42.bpo-28055.f49kfC.rst

```diff
@@ -0,0 +1 @@
+fix unaligned accesses in siphash24()
```

Comment:
Start the sentence from the title case, add missed period, and please add "Patch by *yourname*."